t/vep.t

```diff
@@ -272,7 +274,16 @@ $vep_output =
 }];
 
 $json = json_POST($vep_post,$vep_post_body,'POST a selection of regions to the VEP');
-eq_or_diff($json,$vep_output, "VEP region POST");
+
+cmp_bag($json,$vep_output, "VEP region POST");
+
+$vep_post = '/vep/homo_sapiens/id';
+my $vep_post_body_invalid = '{ "ids" : ["invalid_id", "rs186950277", "rs17081232" ]}';
+$json = json_POST($vep_post,$vep_post_body_invalid,'POST a selection of IDs to VEP');
+
+$vep_post = '/vep/homo_sapiens/hgvs';
+$vep_post_body_invalid = '{ "hgvs_notations" : ["invalid_hgvs", "ENST00000314040:c.311G>T"] }';
+$json = json_POST($vep_post,$vep_post_body_invalid,'POST a selection of HGVS variants to VEP');
```

Comment:
I guess you're testing that faulty input data as the first array value of a POST request will no longer crash, but this should be made explicit as a set of tests.

Reply:
Also I'm confused as to how to consider this in relation to #239. Should we close that and continue with this?